src/UserManager.php

```diff
@@ -184,25 +192,45 @@ public function commitDeletionList(): void {
       $deletedUserIds[] = $userId;
     }
 
-    $this->logger->info($this->formatPlural(
-      count($deletedUserIds),
-      'One user to be deleted',
-      '@count users to be deleted'
-    ));
+    $this->logger->info(
+      '@count users to be deleted',
+           [
+             '%count' => count($deletedUserIds),
+           ]
+    );
+
     if ($this->options['debug'] ?? FALSE) {
       $users = $this->userStorage->loadMultiple($deletedUserIds);
-      foreach ($users as $user) {
-        $this->logger->debug(sprintf('User to be deleted: %s (#%s)', $user->label(), $user->id()));
+      if (empty($users)) {
+
+        $this->logger->info('No users to be deleted');
+
+      }
+      else {
+        foreach ($users as $user) {
+
+          $this->logger->info(
+            'User to be deleted: %user (%id)',
+                 [
+                   '%user' => $user->label(),
+                   '%id' => $user->id(),
+                 ]
+          );
+
+        }
       }
     }
 
     if (!($this->options['dry-run'] ?? FALSE)) {
       if (!empty($deletedUserIds)) {
-        $this->logger->info($this->formatPlural(
-          count($deletedUserIds),
-          'Deleting one user',
-          'Deleting @count users'
-        ));
+
+        $this->logger->info(
+          'Deleting @count user(s)',
+               [
+                 '%count' => count($deletedUserIds),
+               ]
```

Comment:
The indenting is a little inconsistent (I think):

```suggestion
          'Deleting @count user(s)',
          [
            '%count' => count($deletedUserIds),
          ]
```